### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by Ahmedoudouh**

### CODE REVIEW

File: src/footer/button.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./style.css";
+export const Button = ({ icon , name }) => {
```
[the remark this answers is not in the document]
Comment (Ahmedoudouh) [reply src/footer/button.jsx:3]: yes of course

File: src/footer/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    flex: 1;
+    padding: 6px 12px 8px;
+    min-width: 80px;
+    max-width: 168px;
+    color: white;
+    background-color: #2d313a;
+    border: none;
+    font-size: 0.875rem;
+    cursor: pointer;
+    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
```
Comment (faouzioudouh) [inline src/footer/style.css:23]: Why is this font-family here?

File: src/footer/style.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+.cta {
+    flex: 1;
+    padding: 6px 12px 8px;
+    min-width: 80px;
+    max-width: 168px;
+    color: white;
+    background-color: #2d313a;
+    border: none;
+    font-size: 0.875rem;
```
Comment (faouzioudouh) [inline src/footer/style.css:21]: Do not copy past the styles from the website!!!!!

File: src/footer/style.css [hunk 4]
```diff
@@ -13,14 +13,13 @@
...
-    min-width: 80px;
     max-width: 168px;
     color: white;
     background-color: #2d313a;
     border: none;
-    font-size: 0.875rem;
+    font-size: 13px;
     cursor: pointer;
-    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+    font-family: "Roboto";
```
Comment (faouzioudouh) [inline src/footer/style.css:22]: No font family!!!!!!!

File: src/index.js [hunk 5]
```diff
@@ -1,10 +1,16 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { ReactQueryDevtools } from "react-query/devtools";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
+
+const queryClient = new QueryClient();
 root.render(
```
Comment (faouzioudouh) [inline src/index.js:10]: Keep empty line

File: src/singleContinent/singleContinent.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const SingleContinent = () => {
```
Comment (faouzioudouh) [inline src/singleContinent/singleContinent.jsx:3]: What is this?

File: src/trending.jsx [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  const { data, isLoading, isError } = useFetchData();
+  console.log(data);
+  return (
+    <div>
+      <div className="trending">
+        {isLoading && <h1>loading....</h1>}
+        {isError && <h1>try again!....</h1>}
+      </div>
+    </div>
+  );
```
Comment (faouzioudouh) [inline src/trending.jsx:19]: Ahmed, format your code please.

Comment (younessoudouh) [reply src/trending.jsx:19]: could you make a custom hook for useFetchData

**Review by faouzioudouh — APPROVED** [review]

**Pull request merged**